podman_compose.py

```diff
@@ -2341,6 +2341,18 @@ def _published_target(port_string):
             print(published)
             return
 
+@cmd_run(podman_compose, 'pause', 'Pause all running containers')
+def compose_pause(compose, args):
+    containers = list(reversed(compose.containers))
+    for container in containers:
+        compose.podman.run(['pause'], container['name'])
+
+@cmd_run(podman_compose, 'unpause', 'Unpause all running containers')
+def compose_pause(compose, args):
```

Comment:
Sry but is it not a little bit confusing that you named a function `compose_pause` even though the function body unpaused all containers?

Reply:
fixed